lib/generators/avo/templates/locales/avo.nb.yml

```diff
@@ -1,4 +1,4 @@
-nb-NO:
```

Comment:
Is this the right way of having the translations in Norway `nb` vs. `nb-NO`? I'm not very familiar with localization.

Reply:
Yes it is. [Example reference](https://github.com/svenfuchs/rails-i18n/commit/196ea80372e08941f9e9e8899e3af7b0b86e703c).

Reply:
Thank you @tersor! I appreciate the help.

Could we, in the future when we add new strings, ping you to validate, and help with Norwegian translations?

Reply:
Jep, sure!